dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
+COPY ./requirements.txt /dockerBuild/requirements.txt
+WORKDIR /dockerBuild
+RUN pip install -r requirements.txt
+COPY . /dockerBuild
+ENTRYPOINT [ "python" ]
+CMD [ "run.py" ]
```

Comment:
Better replace this command with: explicit mention of host `0.0.0.0` here itself rather than changing the default in `run.py`

Reply:
Done!